### DISCUSSION TITLE
Ignoring Exclusive

### DISCUSSION THREAD

**Discussion by lodmfjord** [wmww/gtk-layer-shell#124] [Q&A] [answered]
I have one app with exclusive, but I want to force draw another app to small corner of the exclusive. 

Is that possible?

**Comment by lodmfjord** [comment, marked as answer]
Oh I just set layer as overlay and had negative margin.

Not the best solution but it works.

**Comment by wmww** [comment]
The underlying Wayland protocol is defined here: https://github.com/wmww/gtk-layer-shell/blob/master/protocol/wlr-layer-shell-unstable-v1.xml, and it's up to specific compositors to implement it correctly. This library strives to expose the full protocol functionality, but can't do anything beyond that. I think there was something in the protocol about negative exclusive zones that might be relevant, but I don't remember the details.
